clap-utils/src/keypair.rs

```diff
@@ -316,7 +337,7 @@ pub fn signer_from_path_with_config(
         legacy,
     } = parse_signer_source(path)?;
     match kind {
-        SignerSourceKind::Prompt => {
+SignerSourceKind::Prompt => {
```

Comment:
How'd you sneak this past `rustfmt`? :slightly_smiling_face:

Reply:
Apparently `rustfmt` has the same blind spot I do :-)